hbase/hbase.sh

```diff
@@ -76,36 +127,34 @@ EOF
       --configuration_file 'hbase-site.xml.tmp' \
       --name 'hbase.zookeeper.quorum' --value "${zookeeper_nodes}" \
       --clobber
-    bdconfig set_property \
-      --configuration_file 'hbase-site.xml.tmp' \
-      --name 'hbase.rootdir' --value "hdfs://${CLUSTER_NAME}-m:8020/hbase" \
-      --clobber
-
-    "${HBASE_HOME}/bin/hbase-daemon.sh" start zookeeper
   fi
 
   bdconfig merge_configurations \
     --configuration_file "${HBASE_HOME}/conf/hbase-site.xml" \
     --source_configuration_file hbase-site.xml.tmp \
     --clobber
-  
-  # On single node clusters we must also start regionserver on it.
+
+  # On single node clusters we must also start regionserver on it and provide zookeeper-server.
   if [[ "${WORKER_COUNT}" -eq 0 ]]; then
-    "${HBASE_HOME}/bin/hbase-daemon.sh" start regionserver
+    install_apt_get zookeeper-server || err 'Unable to install Zookeeper on single node cluster.'
```

Comment:
Let's not special case zookeeper installation on single node cluster. Similar to standard cluster, we can use initiailization action to install zookeeper even on the single node cluster.

Reply:
@aniket486 I agree this was just workaround, so then first we need to modify zookeeper init action, because in [README](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/blob/master/zookeeper/README.MD#important-notes) of zookeeper init action it is mentioned that: 
> This script will not work on Single Node clusters, as it requires at least 3 nodes.

Reply:
I removed this line in latest commits. Now when creating standard or single cluster it is required to additionally pass zookeeper init action. The fix for running zookeeper on single clusters is here: https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/418.